src/server/game/Entities/Object/Object.h

```diff
@@ -1047,9 +1048,11 @@ class WorldObject : public Object, public WorldLocation
 
         bool CanNeverSee(WorldObject const* obj) const;
         virtual bool CanAlwaysSee(WorldObject const* /*obj*/) const { return false; }
-        bool CanDetect(WorldObject const* obj, bool ignoreStealth, bool checkClient) const;
+        //bool CanDetect(WorldObject const* obj, bool ignoreStealth, bool checkClient) const;
+        bool CanDetect(WorldObject const* obj, bool ignoreStealth, bool checkClient, bool checkAlert = false) const;
         bool CanDetectInvisibilityOf(WorldObject const* obj) const;
-        bool CanDetectStealthOf(WorldObject const* obj) const;
+        //bool CanDetectStealthOf(WorldObject const* obj) const;
```

Comment:
Is there a reason to leave this as a comment? If it isn't needed anymore then remove it.